corehq/form_processor/management/commands/list_data_by_shard.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+from collections import Counter
+from django.core.management.base import BaseCommand
+from corehq.form_processor.backends.sql.dbaccessors import ShardAccessor
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors, FormAccessors
+
+
+class Command(BaseCommand):
+    help = "List forms and cases in a domain by shard ID."
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain', help='Domain to create data in.')
+
+    def handle(self, domain, **options):
+        forms_by_shard = Counter()
+        forms_by_db = Counter()
+        cases_by_shard = Counter()
+        cases_by_db = Counter()
+        print('======================== forms ========================')
+        print('id\t\t\t\t\tshard\tdatabase')
+        for form_id in sorted(FormAccessors(domain=domain).get_all_form_ids_in_domain()):
```

Comment:
Maybe not a good idea to load ~70M IDs into memory? (I think it's about 5GB)

Reply:
haha, this is just for local testing, but i agree. was doing a small domain-level analysis

Reply:
I think this command will eventually be deleted/superseded by other things. let me know if you want me to do that now though am tempted to leave it around until I'm a little further along